From 7750a1b8fbc150385510b20d8f8ba09114a3e996 Mon Sep 17 00:00:00 2001
From: Shawn Guo <shawn.guo@freescale.com>
Date: Tue, 17 Jun 2014 14:02:40 +0800
Subject: [PATCH] ENGR00317981: ARM: remove DEBUG_LL from imx_v7_defconfig

The CONFIG_DEBUG_LL should only be enabled for debugging early boot
issue on particular board.  Let's drop it from defconfig.

Note, CONFIG_MTD_M25P80 is dropped by running savedefconfig.

Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index d773659..1edfb6b 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -83,7 +83,6 @@ CONFIG_MTD_CFI_AMDSTD=y
 CONFIG_MTD_CFI_STAA=y
 CONFIG_MTD_PHYSMAP_OF=y
 CONFIG_MTD_DATAFLASH=y
-CONFIG_MTD_M25P80=y
 CONFIG_MTD_SST25L=y
 CONFIG_MTD_NAND=y
 CONFIG_MTD_NAND_GPMI_NAND=y
@@ -298,9 +297,6 @@ CONFIG_MAGIC_SYSRQ=y
 # CONFIG_SCHED_DEBUG is not set
 # CONFIG_DEBUG_BUGVERBOSE is not set
 # CONFIG_FTRACE is not set
-CONFIG_DEBUG_LL=y
-CONFIG_DEBUG_IMX6Q_UART=y
-CONFIG_EARLY_PRINTK=y
 CONFIG_SECURITYFS=y
 # CONFIG_CRYPTO_ANSI_CPRNG is not set
 # CONFIG_CRYPTO_HW is not set
-- 
1.8.0

